api/v1alpha1/linodevpc_types.go

```diff
@@ -34,8 +34,6 @@ type LinodeVPCSpec struct {
 	// +kubebuilder:validation:XValidation:rule="self == oldSelf",message="Value is immutable"
 	// +optional
 	Label string `json:"label,omitempty"`
-	// +optional
-	Description string `json:"description,omitempty"`
```

Comment:
The VPC object has `description` fireld: https://github.com/linode/linodego/blob/0bd0c941538822724dbd5cd23c990c9ac7717b17/vpc.go#L25 why we need to delete it?

Reply:
Ah, thanks! I must have missed that. I thought the field was completely unused.